kii_thing_if.h

```diff
@@ -26,7 +26,9 @@ typedef enum kii_thing_if_error_t {
     /** thing-if ThingSDK is alreday started. */
     KII_THING_IF_ERROR_ALREADY_STARTED,
     /** Length of HTTP request exceeded request buffer. */
-    KII_THING_IF_ERROR_REQUEST_BUFFER_OVER_FLOW
+    KII_THING_IF_ERROR_REQUEST_BUFFER_OVER_FLOW,
+    /** Http error. */
+    KII_THING_IF_ERROR_HTTP
```

Comment:
@satoshikumano 
I think that connection errors such as timeout are socket level error.

What do you think?